src/errorpage.cc

```diff
@@ -698,6 +709,7 @@ ErrorState::ErrorState(err_type t, Http::StatusCode status, HttpRequest * req, c
         httpStatus = ErrorDynamicPages[page_id - ERR_MAX]->page_redirect;
     else
         httpStatus = status;
+    debugs(4, 3, *this);
```

Comment:
Please use syntax required by the `find-alive.pl` script.
```suggestion
    debugs(4, 3, "ErrorState constructed, this=" << this, << ", " << *this);
```

This should also be placed at the end of the constructor to ease future output of the other stored details.

Reply:
`find-alive.pl` hints added to public ErrorState constructors and destructor in branch commit 5f65f2f.